h2o-py/h2o/__init__.py

```diff
@@ -23,7 +23,7 @@
 # one would have to import it from h2o.frames.
 from h2o.frame import H2OFrame  # NOQA
 
-__version__ = "SUBST_PROJECT_VERSION"
+__version__ = "3.11.0.99999"
```

Comment:
This seems like a side-effect from running a build script. Should probably be reverted